packages/flutter/lib/src/widgets/focus_traversal.dart

```diff
@@ -1556,56 +1554,112 @@ class FocusTraversalGroup extends StatefulWidget {
   /// {@macro flutter.widgets.ProxyWidget.child}
   final Widget child;
 
-  /// Returns the focus policy set by the [FocusTraversalGroup] that most
-  /// tightly encloses the given [BuildContext].
+  /// Returns the [FocusTraversalPolicy] that applies to the nearest ancestor of
+  /// the given [FocusNode].
   ///
-  /// It does not create a rebuild dependency because changing the traversal
-  /// order doesn't change the widget tree, so nothing needs to be rebuilt as a
-  /// result of an order change.
+  /// Will return null if no [FocusTraversalPolicy] ancestor applies to the
+  /// given [FocusNode].
+  ///
+  /// The [FocusTraversalPolicy] is set by introducing a [FocusTraversalGroup]
+  /// into the widget tree, which will associate a policy with the focus tree
+  /// under the nearest ancestor [Focus] widget.
+  ///
+  /// This function differs from [maybeOf] in that it only traverses the focus
+  /// tree to determine the policy in effect. Unlike this function, the
+  /// [maybeOf] function first walks up the widget tree, to find the nearest
+  /// ancestor [Focus] or [FocusScope] widget, and then calls this function with
+  /// the focus node associated with that widget to determine the policy in
+  /// effect.
+  ///
+  /// See also:
+  ///
+  /// * [maybeOf] for a similar function that will return null if no
+  ///   [FocusTraversalGroup] widget ancestor is found in the widget tree.
```

Comment:
Maybe remove this "See also" section - its kinda confusing. Also, the previous paragraph just talked at length about `maybeOf`...